packages/components/CHANGELOG.md

```diff
@@ -16,7 +16,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
-- `Spinner`, `Link Pure`, `Button Pure`, `Checkbox Wrapper`, `Radio Button Wrapper` matches new design language
+- `Spinner`, `Link Pure`, `Button Pure`, `Checkbox Wrapper`, `Radio Button Wrapper`, `Inline Notification`, `Banner` matches new design language
```

Comment:
Changing a prop value from `neutral` to `info` is a breaking change which should be mentioned here?